docker-compose.yml

```diff
@@ -1748,8 +1748,6 @@ services:
     volumes: *conda-volumes
     environment:
       <<: [*common, *ccache]
-      ARCHERY_INTEGRATION_WITH_NANOARROW: 0
```

Comment:
Just curious. Should we mark it =1 here and let the script take the passed value?